### ISSUE TITLE
UAP8 Application Extension page, and sub-elements questionable in documentation set.

### ISSUE THREAD

**Issue by TimMangan** [MicrosoftDocs/winrt-related#297]
Looking at the updated documentation (Thank You!) https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-uap8-extension I immediately see some questionable items that either need to be reworked or better explained:

- The syntax of the child item is not proper XML Schema syntax, as in parentheses versus angle brackets.
- Appearing in the Executable and EntryPoint attributes description is this text: "If that EntryPoint property isn't specified, the EntryPoint defined for the app is used".  It seems like the Executable/EntryPoint should just be the EntryPoint of the parent Application element.  If something else needs to be run, it should be placed in a separate Application element.  And what is the difference between an Executable and an EntryPoint anyway?
- A whole slew of the attributes also appear to be more consistent with the already defined attributes of the Application Element. Why are these here?  I'm guessing they shouldn't be.

**Comment by drewbatgit** [comment]
@timmangan Trying to close out some of these old issues. I don't see the syntax issues you're referring to - maybe they were fixed in a previous update. I confirmed that all of the attributes are indeed included in this schema. There are several "attribute groups" added to all of the extensions that brought in all of these attributes. The description for EntryPoint is indeed confusing - although it seems to be consistent across all extensions. I've made a note to follow up on this with the subject matter experts and see if we need to update these descriptions across the docs.
